packages/cli/api-importers/openapi/openapi-ir-to-fern/src/buildEndpoint.ts

```diff
@@ -606,3 +623,15 @@ function getRequest({
         assertNever(request);
     }
 }
+
+function endpointRequestSupportsInlinedPathParameters({
+    context,
+    request
+}: {
+    context: OpenApiIrConverterContext;
+    request: Request | undefined;
+}): boolean {
+    // octet-stream requests do not support named request wrappers,
+    // so we can't inline path parameters for them.
+    return context.inlinePathParameters && request?.type !== "octetStream";
```

Comment:
nit: if we make this a switch it would be great so that folks are forced to update if we add a new type

Reply:
Good call, updated!